code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1313,16 +1316,31 @@
 // Note: For effects on subtypes, use the emp_effect() proc instead
 /obj/item/bodypart/emp_act(severity)
 	var/protection = ..()
-	// If the limb doesn't protect contents, strike them first
-	if(!(protection & EMP_PROTECT_CONTENTS))
-		for(var/atom/content as anything in contents)
-			content.emp_act(severity)
+	if((protection & EMP_PROTECT_WIRES) || !IS_ROBOTIC_LIMB(src))
```

Comment:
I don't think I understand what you modified in this proc or why.

Reply:
isnt this an old change that i fixed?

Reply:
fuck you're probably right

Reply:
er like an accidental revert